### PR TITLE
fix for #21, always create reference

### PR DESCRIPTION
**Pull request by matt-land**
Quick fix for this, ready for review. I do not have a board handy to test with.

### CODE REVIEW

File: adafruit_bitmapsaver.py [hunk 1]
```diff
@@ -38,18 +39,18 @@
...
-    output_file.write(struct.pack("<I", 40))
-    output_file.write(struct.pack("<I", width))
-    output_file.write(struct.pack("<I", height))
-    output_file.write(struct.pack("<H", 1))
-    output_file.write(struct.pack("<H", 24))
+    output_file.write(struct.pack("<I", 40))  # pylint: disable=no-member
+    output_file.write(struct.pack("<I", width))  # pylint: disable=no-member
+    output_file.write(struct.pack("<I", height))  # pylint: disable=no-member
+    output_file.write(struct.pack("<H", 1))  # pylint: disable=no-member
+    output_file.write(struct.pack("<H", 24))  # pylint: disable=no-member
```
Comment (FoamyGuy) [inline adafruit_bitmapsaver.py:53]: Why were these pylint disables added? Was it failing the pylint for you without these?

Comment (FoamyGuy) [inline adafruit_bitmapsaver.py:53]: Why were these pylint disables added? Was it failing the pylint for you without these?

Comment (matt-land) [reply adafruit_bitmapsaver.py:53]: it fails pre-commit hooks
```
pylint (library code)....................................................Failed
- hook id: pylint
- exit code: 2

************* Module adafruit_bitmapsaver
adafruit_bitmapsaver.py:55:0: E0011: Unrecognized file option 'ignore' (unrecognized-inline-option)
adafruit_bitmapsaver.py:48:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:51:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:55:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:56:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:57:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:58:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:59:22: E1101: Module 'struct' has no 'pack' member (no-member)

**Comment by FoamyGuy** [comment]
I did test the functionality of this and confirmed it does resolve #21 successfully. 

I think we can remove the pylint disables unless they are specifically needed for passing CI. I ran pre-commit locally without them and it passed, but perhaps my environment is different or something. Did pylint raise errors for you when you made this PR before those disables were added?

**Comment by matt-land** [comment]
> I think we can remove the pylint disables unless they are specifically needed for passing CI. 

Removed

>Did pylint raise errors for you when you made this PR before those disables were added?
yes. If i checkout main and run, I see the following:

```
 pylint adafruit_bitmapsaver.py 
************* Module adafruit_bitmapsaver
adafruit_bitmapsaver.py:41:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:44:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:48:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:49:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:50:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:51:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:52:22: E1101: Module 'struct' has no 'pack' member (no-member)

------------------------------------------------------------------
Your code has been rated at 5.62/10 (previous run: 5.29/10, +0.33)

**Comment by FoamyGuy** [comment]
we might have some configuration in the .pylintrc file that is making that not get flagged when it runs in CI. Or perhaps it's due to a version difference or something.

**Review by FoamyGuy — APPROVED** [review]
Changes look good to me. Tested successfully on PyPortal with the `bitmapsaver_simpletest.py` from this repo.

Thanks for the fix @matt-land

**Pull request merged**